onadata/libs/filters.py

```diff
@@ -29,7 +29,7 @@ def filter_queryset(self, request, queryset, view):
         """
         if (
             request.user.is_anonymous
-            or OneTimeAuthToken.is_signed_request(request)
+            or OneTimeAuthToken.is_signed_request(request)[0]
```

Comment:
I do wonder if we should rename the method because someone thinking this would return a `bool` would be surprised:
```
In [1]: bool((None,))  # or even bool((False,))
Out[1]: True
```
(I did search through the entire codebase to make sure the method isn't called incorrectly anywhere)

Reply:
Totally agree. Any suggestions?

Reply:
Let's leave it for now. We'll be refactoring in https://github.com/kobotoolbox/kpi/issues/3383